### PR TITLE
Module 1 Capstone Project

### PR DESCRIPTION
**Pull request by see-why**
Add desktop and mobile versions of project, following  [the design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), implementing a home and about page.

Mobile version has a pop-up menu and Program Guest cards are added dynamically via Javascript.

This branch only carries the dynamic Javascript, the rest of the features have been added using a different branch to main, as such all other changes are on the main branch.

[Loom Video](https://www.loom.com/share/db103a6603544d56ace082ee0ad14c72)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,94 +113,11 @@ <h2> SEE THE WHOLE PROGRAM </h2>
                 <button class="register-button" type="button"> Join Lagos Comic-Con 2021 </button>               
             </div>
         </div> 
-        <div class="speakers-div-container"> 
+        <div class="speakers-div-container" id="speakers-div-container"> 
             <header class="speakers-div-container-header">
                 <h2> Guest Appearances </h2>
                 <hr>
```
Comment (Emmyn5600) [inline index.html:120]: #### Mobile Size
It contains an hamburger menu on the left side.

- [ ] The hamburger button on the mobile section should be clickable and display the menu.

- [ ] I suggest to change the font-size for our first heading to look more clear on the mobile section 

cpp14

-  I suggest to change the font-size for our first heading to look more clear on the mobile section 
cpp15

File: scripts/export.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+  undoCollapseButtonDiv.appendChild(undoCollapseButton);
+
+  speakersDivContainer.appendChild(undoCollapseButtonDiv);
+}
+
+const section = document.getElementById('');
+if (section !== null) {
+  section.addEventListener('click', loadSpeakers);
+}
```
Comment (Emmyn5600) [inline scripts/export.js]: - [ ] Great implementation here to use Js file to keep the image and description for the featured speakers however, let us add some decoration to the image.

-----------------------------------------------Yours---------------------------------------

cpp16

------------------------------------------Original---------------------------------------------

cpp17

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @see-why 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- The live link demo is working properly:heavy_check_mark:

Changes required :writing_hand: 
   - The about menu redirect us to the about section, however you need to create About page instead of redirect us to the about section
   #### Page Structure and Navigation
        The project contains a `Home` page and `About` page.

   - Please check the comment under the review.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @see-why 👋🏽
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._👀👇🏽
- As the previous code reviewer said, the project contains 2 pages; the `home page` and the `About page` but you are missing the About page. Please fix that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
#### Hello @see-why. 

Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations! 🎉 👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] You can add the following section in your Readme file to help those that are interested in your project to easily get it set up locally
```
## Getting Started
To get a local copy of the repository please run the following commands on your terminal:
$ git clone https://github.com/...
$ cd ...
$ git checkout ...
$ Open index.html in your browser or with a live server
```

_Good luck with your next project and Happy coding!_ 💻

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by see-why** [comment]
> ## Status: PR Approved 💯 🥇
> #### Hello @see-why.
> Your project is complete! You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR.
> Congratulations! 🎉 👏👏
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> * [x]  You can add the following section in your Readme file to help those that are interested in your project to easily get it set up locally
> 
> ```
> ## Getting Started
> To get a local copy of the repository please run the following commands on your terminal:
> $ git clone https://github.com/...
> $ cd ...
> $ git checkout ...
> $ Open index.html in your browser or with a live server
> ```
> 
> _Good luck with your next project and Happy coding!_ 💻
> 
> Cheers and Happy coding! 💻
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝
> 
> _Code Reviewer **[Jane](https://witahgeorjane.tech/)**_
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

thanks a lot.
I have added it as you suggested.

**Pull request merged**